test/features/can_search_collection_test.rb

```diff
@@ -42,14 +42,22 @@ class CanSearchCollectionTest < SystemTestCase
           'User-Agent'=>'Ruby'
         }).
       to_return(status: 200, body: get_stub("search_collection_C1996546695-GHRC_DAAC.atom"), headers: {})
-    stub_request(:get, "https://cmr.sit.earthdata.nasa.gov/search/concepts/C1996546695-GHRC_DAAC.echo10").
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/concepts/C1996546695-GHRC_DAAC.echo10").
       with(
         headers: {
           'Accept'=>'*/*',
           'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
           'User-Agent'=>'Ruby'
         }).
       to_return(status: 200, body: get_stub("C1996546695-GHRC_DAAC_echo10.xml"), headers: {})
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.umm_json?collection_concept_id=C1996546695-GHRC_DAAC&page_size=10&page_num=1").
+      with(
+        headers: {
+          'Accept'=>'*/*',
+          'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+          'User-Agent'=>'Ruby'
+        }).
+      to_return(status: 200, body: '{"hits" : 0, "took" : 105, "items" : []}', headers: {})
   end
   describe 'search cmr collection' do
```

Comment:
Maybe this should be called, search and ingest a collection,   And have a test for ingesting a collection with echo10 granule and another for a umm json record.